src/Symfony/Component/Config/Loader/FileLoader.php

```diff
@@ -85,9 +85,13 @@ public function import($resource, $type = null, $ignoreErrors = false, $sourceRe
     {
         $ret = array();
         $ct = 0;
-        foreach ($this->glob($resource, false, $_, $ignoreErrors) as $resource => $info) {
-            ++$ct;
+        if (!is_string($resource) || false === strpbrk($resource, '*?{[')) {
             $ret[] = $this->doImport($resource, $type, $ignoreErrors, $sourceResource);
+        } else {
+            foreach ($this->glob($resource, false, $_, $ignoreErrors) as $path => $info) {
```

Comment:
and I know this is not what this PR tries to improve. But I detected it while reviewing it. And this looks like one of the cases that we still need to polish before the stable release including glob support.